airflow/models/variable.py

```diff
@@ -152,6 +152,11 @@ def get(
                 mask_secret(var_val, key)
                 return var_val
 
+    @staticmethod
+    @provide_session
+    def list(session: Session = None) -> list[Variable]:
```

Comment:
i'm not sure we want or need a method to list _all_ variables without any filtering. why is this needed?  i'm guessing you use it in a test or something.... if that's true, you should just write a query in the test.

Reply:
Ok, removed.